### PR TITLE
refactor(test): Modernize the fx_fennec_v1_sign_in functional suite.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Use ES2015 destructuring and selectors.js

Extraction from #5408

@mozilla/fxa-devs - r?

### CODE REVIEW

File: tests/functional/fx_fennec_v1_force_auth.js [hunk 1]
```diff
@@ -5,31 +5,35 @@
...
+    createUser,
+    fillOutForceAuth,
+    fillOutSignInUnblock,
+    openForceAuth,
+    openVerificationLinkInDifferentBrowser,
+    openVerificationLinkInNewTab,
+    respondToWebChannelMessage,
+    testElementExists,
+    testIsBrowserNotified,
+    thenify,
```
Comment (vbudhram) [inline tests/functional/fx_fennec_v1_force_auth.js]: nit: remove extra `,`

File: tests/functional/fx_fennec_v1_sign_in.js [hunk 2]
```diff
@@ -18,26 +18,27 @@ define([
...
+    openPage,
+    openVerificationLinkInDifferentBrowser,
+    openVerificationLinkInNewTab,
+    respondToWebChannelMessage,
+    testElementExists,
+    testElementTextEquals,
+    testElementTextInclude,
+    testIsBrowserNotified,
+    thenify,
+    type,
```
Comment (vbudhram) [inline tests/functional/fx_fennec_v1_sign_in.js]: remove `,`

File: tests/functional/fx_fennec_v1_force_auth.js (hunk 1 above)
Comment (shane-tomlinson) [reply tests/functional/fx_fennec_v1_force_auth.js]: I'm going to make a proposal: We keep the trailing `,` where it's allowed by JavaScript, doing so makes it so someone can add lines afterwards without affecting the history of this line.

Comment (vbudhram) [reply tests/functional/fx_fennec_v1_force_auth.js]: I was going to make counter proposal to add it as a lint rule :) I don't mind either way as long as everything is consistent.

Comment (shane-tomlinson) [reply tests/functional/fx_fennec_v1_force_auth.js]: > add it as a lint rule

I like this plan.

**Pull request merged**